eos-firewall-localonly.service

```diff
@@ -0,0 +1,12 @@
+# Run eos-firewall-localonly before NetworkManager starts
+
+[Unit]
+Description=Endless local-only firewall
+Before=NetworkManager.service
+
+[Service]
+Type=oneshot
```

Comment:
Historical accident as I only added start/stop syntax after I wrote the systemd unit. Fixed with ExecPostStop as discussed on Slack, which also cleans up in the case of a failed start.